recipe/meta.yaml

```diff
@@ -73,7 +70,7 @@ requirements:
     - netcdf-cxx-legacy  # [tempest == 'tempest']
```

Comment:
we shouldn't need these anymore. Tempest distributes its own NetCDF-C++ headers and implementation that is frozen at the last legacy version. So this is no longer an explicit dependency. Just FYI